README.MD

```diff
@@ -1,224 +1,277 @@
+### What happens if I uninstall the helm chart?
+The volumes will still retain the data and if you install the chart again to the same namespace you'll be able to access it.
 
-### Add table to cluster:
+### What Is the Balancer and How Does It Work?
 
-| SQL | HTTP | 
-|---|---|
-|`ALTER CLUSTER manticore_cluster ADD idx;`|`curl -X "POST" localhost:7308/sql -d "mode=raw&query=alter cluster manticore_cluster add idx"`|
+The balancer in ManticoreSearch is a key component for managing high availability (HA) in a replication cluster. It ensures that read operations are distributed efficiently across available worker pods, while write operations bypass the balancer and are sent directly to the worker pods.
 
-### Add some data to your table:
+#### Key Functions of the Balancer:
+- **Read Operations:** The balancer intelligently routes read queries (`SELECT`) to the most appropriate workers based on their status and availability. This ensures load is distributed evenly across active workers, enhancing cluster efficiency and reliability.
+- **Write Operations:** Writes (`INSERT`, `UPDATE`, etc.) should be manually sent directly to the worker pods, bypassing the balancer.
 
-**SQL**
-```sql
-INSERT INTO manticore_cluster:idx (title) VALUES ('dog is brown'), ('mouse is small');
-```
+#### How the Balancer Works:
+1. **Dynamic Synchronization:**
+   - If `balancer.enabled` is set to `true` (default), the balancer checks the table configuration on the first active worker every 5 seconds (configurable via the `balancer.runInterval` parameter).
+   - If the configuration differs from its current state, the balancer reloads itself to align with the updated tables.
 
-**HTTP**
-```bash
-curl -X "POST" -H "Content-Type: application/x-ndjson" localhost:7308/bulk -d '
+2. **Event-Driven Updates:**
+   - When a worker starts or stops, it sends a signal to the balancer, prompting immediate synchronization without waiting for the interval.
 
-{"insert": {"cluster":"manticore_cluster", "index" : "idx", "doc": { "title" : "dog is brown" } } }
-{"insert": {"cluster":"manticore_cluster", "index" : "idx", "doc": { "title" : "mouse is small" } } }'
-```
+3. **High Availability Strategy:**
+   - The balancer achieves HA by using `index_ha_strategy: nodeads`, ensuring queries are routed only to healthy nodes.
 
-### Forward load balancer's port to your local machine:
+4. **Replica Redundancy:**
+   - To further improve HA for read operations, you can increase the number of balancer replicas by adjusting the `balancer.replicaCount` parameter in `values.yaml`.
 
-```bash
-export NAMESPACE=manticore
-export BALANCER_NAME=$(kubectl get pods --namespace $NAMESPACE \
--l "app.kubernetes.io/name=manticoresearch,app.kubernetes.io/instance=manticore,name=manticore-manticoresearch-balancer" \
--o jsonpath="{.items[0].metadata.name}")
-```
+#### Configuration Options:
+- **Custom Behavior:** You can customize the balancer’s behavior using its HA strategy (`index_ha_strategy`).
```

Comment:
Give a link to the docs with more info about the strategies

Reply:
I completed it in a few lines below